VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
 	src/VersionControlUIException.cpp
 	src/VersionControlUIPlugin.cpp
+	src/DiffManager.cpp
 	test/SimpleTest.cpp
 )
-envision_plugin(VersionControlUI InteractionBase)
+envision_plugin(VersionControlUI InteractionBase oomodel)
```

Comment:
I see. So a better **temporary** solution is not include these dependencies at all. They are not really needed unless you want to compile on Windows or unless you need the dependencies during initialization.

I would remove the dependencies and put a `//TODO Remove this/make it more abstract` right where you use `Class` and add an extra comment explaining that we shouldn't have a dependency on OOModel.